server/game/sprites/escalator.js

```diff
@@ -0,0 +1,43 @@
+const { Settings, SpriteType } = require('../enums')
+
+class Escalator extends Phaser.Physics.Arcade.Sprite {
+  constructor(scene, entityID, x, y) {
+    super(scene, x, y, '')
+    this.scene = scene
+
+    scene.add.existing(this)
+    scene.physics.add.existing(this)
+
+    this.setCollideWorldBounds(true)
+
+    this.type = SpriteType.ESCALATOR
+    this.entityID = entityID
+    this.body.setSize(160, 35)
+    this.body.allowGravity = false
+    this.setImmovable(true)
+    this.velocity = 100
+    this.movingDown = false;
+    this.escalatorPadding = Settings.LEVEL_HEIGHT * 0.25
+
+    scene.events.on('update', this.update, this)
+  }
+
+  update() {
+    if (this.y < this.escalatorPadding && !this.movingDown) {
```

Comment:
Can you describe a bit the logic in the update. I'm not sure I fully understand the escalator padding.